package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bonniernews/local-esi",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
hm, version should be bumped on publish, not PR. 
also, maybe patch rather than minor..